bundles_sort_test.go

```diff
@@ -0,0 +1,95 @@
+package versionbundle
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestBundlesSortByVersion(t *testing.T) {
```

Comment:
I am okay with the test but it feels like you are mainly testing against the coreos library here.
It does improve confidence though :D

Reply:
No the thing here was to first create failing test based on assumptions from "execute code in the head" and then getting it green with changed behavior.